resources/web/docs_js/index.js

```diff
@@ -170,6 +170,34 @@ function init_toc(lang_strings) {
     });
 }
 
+function highlight_otp() {
+  let visibileHeadings = []
+  const observer = new IntersectionObserver(entries => {
+    entries.forEach(entry => {
+      const id = entry.target.getAttribute('id');
+      let element = document.querySelector(`#sticky_content #this_page a[href="#${id}"]`);
+      let itemId = $(element).parent().attr('id')
+      if (entry.intersectionRatio > 0){
+        visibileHeadings.push(itemId);
+      } else {
+        const position = visibileHeadings.indexOf(itemId);
+        visibileHeadings.splice(position, position + 1)
```

Comment:
```suggestion
        visibileHeadings.splice(position, 1)
```

The second argument to `splice` is how many elements to delete. This code is working because, as the heading scrolls off the page any following elements in the array will also be off the page, so removing `position + 1` elements overlaps conceptually. Still, better to remove just the intended element.